Doc/library/functions.rst

```diff
@@ -1625,7 +1625,7 @@ are always available.  They are listed here in alphabetical order.
    not found in statically compiled languages or languages that only support
    single inheritance.  This makes it possible to implement "diamond diagrams"
    where multiple base classes implement the same method.  Good design dictates
-   that this method have the same calling signature in every case (because the
```

Comment:
I don't think this is a typo. "have" makes sense here.

Consider:

> It ought to have the same signature

vs

> It ought to has the same signature

Reply:
My fault, it was indeed correct. I am updating the PR. Thanks for the review @ammaraskar.

Reply:
FYI @brandtbucher.